devika.py

```diff
@@ -120,11 +120,17 @@ def get_agent_state():
     return jsonify({"state": agent_state})
 
 
+# Security Update!!
 @app.route("/api/get-browser-snapshot", methods=["GET"])
 @route_logger(logger)
 def browser_snapshot():
+    # TO-DO: Update allowed paths
```

Comment:
can you finish this TODO?

Reply:
I actually comment out for you guys probably you should give /tmp folder  only bc attacker should not be able to access another directories.